src/test/java/edu/kis/powp/command/SetPositionCommand.java

```diff
@@ -0,0 +1,17 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class SetPositionCommand implements DriverCommand {
+    int x, y;
+    Job2dDriver driver;
+    public SetPositionCommand(int x, int y, Job2dDriver driver){
```

Comment:
It would be good to give a possibility to select the driver before executing the command (more flexibility), e.g. introducing a setter.